From ea336ba22c9f2bc52f2b6b2eefec4840e8ede256 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 11 May 2017 16:23:56 +0800
Subject: [PATCH 1767/5242] MLK-14885 dma: pxp: fix potential multi-definition
 issue

commit  ed1e9196f8fb0e3036aad2e3a40ebe588caa91f8 from
https://source.codeaurora.org/external/imx/linux-imx.git

When 'CONFIG_MXC_PXP_CLIENT_DEVICE' disabled, the
'register_pxp_device' and 'unregister_pxp_device'
may cause multiple definitions compiling error.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/pxp_dma.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/pxp_dma.h b/include/linux/pxp_dma.h
index 61e38a7..a48871cae 100644
--- a/include/linux/pxp_dma.h
+++ b/include/linux/pxp_dma.h
@@ -67,8 +67,8 @@ void pxp_txd_ack(struct dma_async_tx_descriptor *txd,
 int register_pxp_device(void);
 void unregister_pxp_device(void);
 #else
-int register_pxp_device(void) { return 0; }
-void unregister_pxp_device(void) {}
+static int register_pxp_device(void) { return 0; }
+static void unregister_pxp_device(void) {}
 #endif
 void pxp_fill(
         u32 bpp,
-- 
1.7.9.5

